src/status_im2/common/bottom_sheet/style.cljs

```diff
@@ -44,14 +46,27 @@
    :top              0
    :bottom           0})
 
+(defn sheet-content
+  [theme padding-bottom-override insets]
+  {:position                :absolute
+   :background-color        (colors/theme-colors colors/white colors/neutral-95 theme)
+   :bottom                  0
+   :left                    0
+   :right                   0
+   :border-top-left-radius  20
+   :border-top-right-radius 20
+   :padding-bottom          (or padding-bottom-override (+ (:bottom insets) bottom-margin))})
+
 (defn selected-item
-  [theme window-height sheet-height {:keys [top]}]
+  [theme top sheet-height show-bottom-margin border-radius]
   {:position          :absolute
-   :bottom            10
-   :max-height        (- window-height sheet-height top)
+   :top               (when (not show-bottom-margin) (- 0 top))
+   ;; Bottom margin of 8 is added when the selected item height is less than
```

Comment:
This comment seems a bit out of place to me. Here 8 is added only when show-bottom-margin is truthy, that's all this function cares about.

But the important feedback I have about this code is different. The number 8 seems to be the same one used on `status-im2.common.bottom-sheet.view/f-view`. This duplication between unrelated pieces of code is very easy to miss, and can lead to future bugs if a developer changes one calculation, but doesn't change the other.

```clojure
(let [...
      show-bottom-margin (< @item-height (- window-height @sheet-height (:top insets) 8))
      ...]
  ...)
```

@jo-mut, my suggestion is to extract 8 into a constant and reuse it in both places.

I have the impression calculations with duplicated magical numbers are one of the very common sources of bugs in the app, the codebase ends up being fragile to trivial changes.